packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -73,6 +85,7 @@ const InternalHeaderCell = <T extends TableItem>({
       <StyledFlex alignItems="center" flexDirection="row" hide={hide} align={align}>
         {children}
         {!hide && renderSortIcon()}
+        {tooltip && renderTooltip()}
```

Comment:
I think it be valuable to check not only that the tooltip is present but that it's also of type `string` before rendering it as-is (perhaps in `renderTooltip`, you can render `null` if not a string)

Reply:
@jordan-massingill  Thanks, done)